data/human/deep missions.txt

```diff
@@ -2431,3 +2431,320 @@ event "battle in arneb end"
+			`	Thorleif turns to you, his face flashing red, and speaks through clenched teeth, "I have no father. I have no memories of such a man, and I will make no effort to remember someone who didn't stick around long enough to know me. Now, go to your rooms and leave me be!"`
+			label rooms
+			`	You quietly follow Helm to two small, sparsely decorated rooms, clearly intended for guest use. Helm clears his throat, "This may surprise you, but that actually went better than I thought it would."`
+			`	Helm then turns his attention to a small window and the darkening horizon beyond it. "Dark clouds are gathering, I fear we may be here longer than a single night."`
+			choice
+				`	(You head to bed as the first rains begin to patter against the roof.)`
+			`	You awake to riotous rain pelting hard against the walls. As Helm predicted a powerful storm blew in over the night, and the floating island found itself besieged by savage weather. By midmorning the walls are starting to creak as the howling wind bashes against it unceasingly.`
+			`	You hear a door open and struggle to close. Then the sound of heavy sloshing footsteps. Thorleif, soaked to the bone, stands before you and Helm in the main room, "It's a Fimbulhrithr," Thorleif exclaims, "One of the worst I've yet seen. If we do not act quickly it could well flatten everything on this isle. Will you help me?"`
+			choice
+				`	(You offer to help Thorleif.)`
+					goto help
+				`	("No, I am done too much for this family already. Especially for free!"`
+			label refuse
+			`	Thorleif growls, "Knew better than to trust a friend of that useless old man. I gave you a place to sleep, and now you won't lift a hand to save it? I should have expected no less!" Thorleif and Helm hurry off to tend to the island during the storm, leaving you warm and cozy and idle inside.`
+			`	Thorleif and Helm heoricially manage to steer the island away from the storm, and to calmer safer seas. However, with only the two of them many of the out-buildings are destroyed along with a few of the small watercraft Thorleif used to fish and work on.`
+			`	You get the impression that while Thorleif is not ruined, this is an expensive and major loss for him. The moment the winds are calm, he demands Helm take you back to the mainland, and your ship. "This Captain," Thorleif's voice oozing with disgust, "is just as lucky as my so-called father. I want him gone."`
+			`	For most of the trip back to your ship Helm says nothing, he has been cool to you ever since you refused to help save the island. Finally, as you are preparing to disembark from his ship he stops you. "You've done Skaldgar no favors here. I'd appreciate it if we don't see from you again."`
```

Comment:
```suggestion
			`	For most of the trip back to your ship, Helm says nothing. He has been cool to you ever since you refused to help save the island. Finally, as you are preparing to disembark from his ship, he stops you. "You've done Skaldgar no favors here. I'd appreciate it if we don't see from you again."`
```